docs/analytics/resources/vars-analytics.yml

```diff
@@ -131,7 +132,7 @@ JENKINS_ANALYTICS_CREDENTIALS:
 # See AWS_analytics_setup.md#jenkins-seed-job
 ANALYTICS_SCHEDULE_SECURE_REPO_URL: "git@github.com:xxx/secure-configuration.git"
 ANALYTICS_SCHEDULE_SECURE_REPO_VERSION: "master"
-ANALYTICS_SCHEDULE_JOBS_DSL_REPO_URL: "git@github.com:edx-ops/edx-jenkins-job-dsl.git"
+ANALYTICS_SCHEDULE_JOBS_DSL_REPO_URL: "git@github.com:edx/jenkins-job-dsl.git"
```

Comment:
This is not the correct repo URL.  Unfortunately, the repo is still private  (cf [jenkins.md#jenkins-seed-job](https://github.com/UmanShahzad/openedx-deployment/blob/cb31b9c97881044a15b5bcdd82be3be079e47206/docs/analytics/jenkins.md#jenkins-seed-job)), and so returns a 404 if you don't have access.

But it has been moved to `git@github.com:edx/jenkins-job-dsl-internal.git, if you could please update it to that.